packages/core/src/Modifiers.ts

```diff
@@ -71,10 +75,31 @@ export class Modifiers {
             return this._contents.find(modifier) as T;
         }
     }
+    /**
+     * Return the first modifier in the array that is an instance of the given
+     * modifier class or create one, append it and return it.
+     * If the modifier passed is a modifier instance, return it if it was
+     * present in the array.
+     *
+     * @param modifier
+     */
+    get<T extends Modifier>(modifier: T | Constructor<T>): T {
+        let got = this.find(modifier);
+        if (got) {
+            return got;
+        } else if (isConstructor<typeof Modifier>(modifier, Modifier)) {
+            got = new modifier();
+            this.append(got);
+            return got;
+        }
```

Comment:
If `found` (`got`) is a `let` anyway then no need for the `if/else` and double `return`.
```typescript
let found = this.find(modifier);
if (!found && isConstructor<typeof Modifier>(modifier, Modifier)) {
    found = new modifier();
    this.append(found);
}
return found;
```